### DISCUSSION TITLE
Compatibility with Windhawk Mods

### DISCUSSION THREAD

**Discussion by KVFX-II** [rocksdanister/lively#2312] [Ideas]
Hey there. I was wondering if in a future patch we could make lively wallpapers work with windhawk mods? Some of the mods are pretty cool but everytime you put them both, they crash. Just an idea. Also, how's the progress on the gallery?

**Comment by rocksdanister** [comment]
What types of mod are causing issue?

**Reply by KVFX-II** [reply]
Specifically when I combine the rain live wallpaper in lively and the "Windows 11 Start Menu Styler" and "Windows 11 Taskbar Styler" in Windhawk. Is it just because I'm using too many mods at the same time?

**Reply by rocksdanister** [reply]
Did you get any error message?

**Reply by KVFX-II** [reply]
Yeah. "Lively has encountered a fatal error. Windows Explorer restarted twice in the last 30 seconds! This may be a conflict between Lively and other desktop customization software or Windows. To avoid further issues, all wallpapers are terminated."

**Reply by rocksdanister** [reply]
Edit [Settings.json](https://github.com/rocksdanister/lively/wiki/Getting-Started#storage)

```json
  "TaskbarCrashTimeOutDelay": 30
  ```
from 30 to 0 or -1 and restart Lively.

**Reply by KVFX-II** [reply]
Ok. Where would I go to change that setting cause I've been looking for it?

**Reply by rocksdanister** [reply]
Where did you check? the wiki link I posted has the paths listed.

**Reply by KVFX-II** [reply]
Ok. I found it. I'll give it a try. Thanks!
